### PR TITLE
add UIKit MKMapView to ImageViewer

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**
Issue: https://github.com/JetBrains/compose-multiplatform/issues/2840

### CODE REVIEW

**Comment by dima-avdeev-jb** [comment]
> Would it be hard to support basic touch events (drag, zoom)? Would help illustrate that this _really_ is a map view and not just an image!

I will try to do it in next PR

File: experimental/examples/imageviewer/shared/src/iosMain/kotlin/example/imageviewer/view/LocationVisualizer.ios.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            val mkMapView = MKMapView()
+            val cityAmsterdam = CLLocationCoordinate2DMake(52.3676, 4.9041)
+            mkMapView.setRegion(
+                MKCoordinateRegionMakeWithDistance(
+                    centerCoordinate = cityAmsterdam,
+                    5000.0, 5000.0
+                ),
+                animated = false
+            )
+            mkMapView.addAnnotation(MKPointAnnotation(cityAmsterdam, "I am here", null))
```
Comment (pjBooms) [inline experimental/examples/imageviewer/shared/src/iosMain/kotlin/example/imageviewer/view/LocationVisualizer.ios.kt:26]: Do we need "I am here" label? Just a pointer should be enough

**Review by pjBooms — APPROVED** [review]

File: experimental/examples/imageviewer/shared/src/androidMain/kotlin/example/imageviewer/view/LocationVisualizer.android.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import androidx.compose.foundation.Image
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+import org.jetbrains.compose.resources.painterResource
+
+@OptIn(ExperimentalResourceApi::class)
+@Composable
+internal actual fun LocationVisualizer(modifier: Modifier) {
```
[the remark this answers is not in the document]
Comment (dima-avdeev-jb) [reply experimental/examples/imageviewer/shared/src/androidMain/kotlin/example/imageviewer/view/LocationVisualizer.android.kt:12]: Yeah, it's not so hard. But at first I want to fully compete necessary functionality on iOS.

File: experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/view/LocationVisualizer.desktop.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import androidx.compose.foundation.Image
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+import org.jetbrains.compose.resources.painterResource
+
+@OptIn(ExperimentalResourceApi::class)
+@Composable
+internal actual fun LocationVisualizer(modifier: Modifier) {
```
[the remark this answers is not in the document]
Comment (dima-avdeev-jb) [reply experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/view/LocationVisualizer.desktop.kt:12]: Yes, in the future this code will differ.

File: experimental/examples/imageviewer/shared/src/iosMain/kotlin/example/imageviewer/view/LocationVisualizer.ios.kt (hunk 1 above)
Comment (dima-avdeev-jb) [reply experimental/examples/imageviewer/shared/src/iosMain/kotlin/example/imageviewer/view/LocationVisualizer.ios.kt:26]: Removed label

**Review by pjBooms — APPROVED** [review]

**Pull request merged**